Cargo.toml

```diff
@@ -19,8 +22,20 @@ sha3 = "0.10.0"
 bincode = "1.3.3"
 rand = "0.8.5"
 thiserror = "1.0"
+num-rational = "0.2"
+num-traits = "0.2"
 num-bigint = "0.4"
 num-integer = "0.1"
+schnorrkel = "0.10.2"
+async-trait = "0.1.52"
+secrecy = "0.8.0"
+futures-timer = "3.0.2"
+log = "0.4.14"
+merlin = "3.0.0"
+rocksdb = "0.18.0"
```

Comment:
`rocksdb` is a quite big dependency. I'm trying to keep the number of crates low (Too compile/develop faster). That's why I have introduced a `node` feature. I disable it when I develop crypto stuff. Maybe you can also make `rocksdb` optional and move it to `node` feature, and use a RAM based database backend while development.